resources/assets/js/components/Application/MyFit/MyFit.tsx

```diff
@@ -117,10 +116,12 @@ export const MyFit: React.FunctionComponent<MyFitProps> = ({
         />
       </p>
       {jobQuestions.map((question, index) => {
-        formRefs.current = [
-          ...formRefs.current,
-          formRefs.current[index] || createRef(),
-        ];
+        if (!formRefs.current.includes(formRefs.current[index])) {
+          formRefs.current = [
+            ...formRefs.current,
+            formRefs.current[index] || createRef(),
+          ];
+        }
```

Comment:
This doesn't seem quite right. The if statement asks is `current` includes an element of `current` - looks like circular logic. And I'm not sure its safe to use index as the key. If a question is deleted, the same index may later refer to a different question. Maybe using question.id would be better?